### ISSUE TITLE
NVDA cannot move mouse pointer to current navigator object in list view in the Reaper

### ISSUE THREAD

**Issue by cary-rowen** [nvaccess/nvda#13440]
### Steps to reproduce:
1. Open Reaper6.51(Version is not the most important);
2. Press F4 to open the Actions window
3. Press the tab key to navigate to the list1 list view;
4. Select any item in the list view and press the Application key to popup the menu and open "Edit selected custom action/ReaScripte";
5. You can then use the Tab key to find the List 1 and List 2;
6. Please press NVDA +numpad4/6 in List 1 or List 2 to navigate the list item;
7. Press NVDA +Numpad Divide in any list item;

### Actual behavior:
Perform step 7 NVDA does not move the mouse pointer to the current navigator object but somewhere else

### Expected behavior:
Perform step 7 and the mouse pointer will be moved to the current navigator object

#### NVDA installed/portable/running from source:
installed
#### NVDA version:
2021.3.3
#### Windows version:
Windows 10 21H2 (x64) build 19044.1526
#### Name and version of other software in use when reproducing the issue:
[Reaper6.51](https://www.reaper.fm/download.php)

#### Other information about your system:
None
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
alpha-24896, The problem can also be reproduced
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by seanbudd** [comment]
Hi @cary-rowen - have you reported this to the Reaper team? Could you do so?

**Comment by josephsl** [comment]
CC @JCSTeh

**Comment by jcsteh** [comment]
How are you determining that the mouse does not move to the correct place? Do you have some sight/sighted assistance? The reason I ask is that NVDA will attempt to move the mouse to the centre of the item, but the centre might not be what you need, and that might be the cause of your issue rather than an NVDA bug.

I do vaguely recall that we had trouble with DPI scaling in list views, but I'm not sure if that's since been fixed.

**Comment by cary-rowen** [comment]
@jcsteh @seanbudd 
Hi, Maybe I'm wrong,  the reason I say this is that I found that my navigator object changed after I pressed nvda+numpadDivide,
for example:
My current navigator object should be the "Custom: Edit marker at cursor" list item in list 1, Then I press NVDA+numpadDivide, nvda report: "List2",
To confirm who the current navigator object is, I press nvda+numpad5, nvda report: "List2listwith 2 rows and 1 columns".

Just to add that I have pressed NVDA+5 and NVDA +6 to make sure the following two settings are on:
- focus moves navigator object on
- caret moves review cursor on

NVDA doesn't fix the DPI bug, but I've tried setting "Scale & Layout" to 100% in windows System Settings > Display Settings.

But anyway, the function of NVDA+numpadDivide is to move the mouse pointer to the current navigator object, so I think the above behavior is incorrect.

thanks

**Comment by cary-rowen** [comment]
cc @jcsteh @seanbudd 
I think more that this might be a bug of NVDA, but I'm not so sure, does this need to be reported by the raper team?

**Comment by jcsteh** [comment]
It's unlikely to be a bug in REAPER; I believe it's a standard
SysListView32.

**Comment by seanbudd** [comment]
I have tested this with #13254 and Reaper v6.66, it appears that Reaper is not DPI aware when using multiple monitor.

Once opening reaper in one monitor, than moving it to a separate montior, reaper does not readjust it's DPI. Visual highlight and mouse tracking continue to not work.

I suggest that this needs to be reported to the Reaper developers.
Let them know that Reaper does not support multiple monitor DPI awareness.
This Windows guide may be of assistance to them: https://docs.microsoft.com/en-us/windows/win32/hidpi/high-dpi-desktop-application-development-on-windows#per-monitor-and-per-monitor-v2-dpi-awareness

**Comment by cary-rowen** [comment]
Yes, I've never used multiple monitors and my scaling standard is also set to 100%

**Comment by seanbudd** [comment]
I cannot reproduce @cary-rowen's original issue with Reaper v6.66 and NVDA 2022.2.1

@cary-rowen, can you confirm this is fixed in the latest Reaper and NVDA?

**Comment by cary-rowen** [comment]
I can still reproduce this using reaper 6.66 with the latest NVDA2022.4Alpha.

**Comment by Adriani90** [comment]
@cary-rowen Is this still an issue in Reaper 7.27 with NVDA 2024.4.1?

**Comment by Adriani90** [comment]
We didn't get any updates from the initial reporter since more than 2 years, closing as abandoned. If you are still having this issue, please comment and we can reopen.
